### PR TITLE
Add release process explanation

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
cc @EPashkin

### CODE REVIEW

File: release_process.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Release process
+
+Since gtk-rs has multiple crates which have inter-dependencies, it's a bit painful to make a new release. So here are the multiple steps:
+
+ * Merge `master` branch into `crate` branch in every repository (starting with `sys` and ending with `gtk`). (/!\ `cairo-sys` is in `cairo` repository, not `sys` /!\)
+ * Update crate version in `Cargo.toml` and the number of its `gtk-rs` dependencies.
+ * Open pull requests to `pending` branches to corresponding repositories.
```
Comment (EPashkin) [inline release_process.md]: `master` ?

Comment (EPashkin) [inline release_process.md]: No, somewhat wrong, need be `crate` to `crate`

**Comment by EPashkin** [comment]
May be needed add "Merge 'pending' branch to 'master' of https://github.com/gtk-rs/examples and add PR"
All other seems fine.

File: release_process.md (hunk 1 above)
Comment (GuillaumeGomez) [inline release_process.md]: To `crate` actually. :p

**Comment by GuillaumeGomez** [comment]
Good point! Btw, do you mind opening the PR from pending to master please? I'm ending my work on gir.

**Comment by EPashkin** [comment]
K

**Comment by GuillaumeGomez** [comment]
Thanks!

**Comment by GuillaumeGomez** [comment]
Ok, should be good this time.

File: release_process.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+Since gtk-rs has multiple crates which have inter-dependencies, it's a bit painful to make a new release. So here are the multiple steps:
+
+ * Merge `master` branch into `crate` branch in every repository (starting with `sys` and ending with `gtk`). (/!\ `cairo-sys` is in `cairo` repository, not `sys` /!\)
+ * Update crate version in `Cargo.toml` and the number of its `gtk-rs` dependencies.
+ * Open pull requests to `crate` branches to corresponding repositories.
+ * Once pull requests have been merged, publish all crates to `crates.io` (using the `cargo publish` command).
+ * Generate new docs. To do so, go to `gtk` directory (in `crate` branch) and run `cargo doc --features=embed-lgpl-docs`. Then run `cp -r target/doc/* ../docs` (where `docs` is the corresponding directory for the `gtk-rs/docs` repository). Commit then push the changes to the `docs` (make a pull request and all the usual stuff...).
+ * Merge `pending` branch into `master` in the `gtk-rs/examples` repository (by opening a pull request of course).
+ * Write a blog post (add the file into `_posts` folder in `gtk-rs.github.io` repository) announcing the new release.
```
Comment (EPashkin) [inline release_process.md]: forgot about update badges

Comment (GuillaumeGomez) [inline release_process.md]: Oh damn, you're absolutely right!

Comment (GuillaumeGomez) [inline release_process.md]: Ah no need. It targets last version every time.

Comment (EPashkin) [inline release_process.md]: But it show old version on badges without fix.

Comment (GuillaumeGomez) [inline release_process.md]: Then I think we don't speak about the same thing. To which badges are you referring to precisely please?

Comment (EPashkin) [inline release_process.md]: On main page `gtk-rs` 
```
Crates
cairo-rs v0.1.2 gdk v0.5.2 gdk-pixbuf v0.1.2 gio v0.1.2  glib v0.1.2 gtk v0.1.2 pango v0.1.2

Announcements 
```

Comment (GuillaumeGomez) [inline release_process.md]: As far as I can tell, it is the last versions: https://crates.io/crates/gtk

Comment (EPashkin) [inline release_process.md]: Its only after updating versions in https://github.com/gtk-rs/gtk-rs.github.io/commit/5822140f5e333b54a0c0b1014933c4d2312f6490

Comment (GuillaumeGomez) [inline release_process.md]: Oh damn, I totally forgot... Sorry about that, you're absolutely right.

**Comment by GuillaumeGomez** [comment]
Do you see anything else?

**Comment by EPashkin** [comment]
May be we need write that PRs on 'crate` branch don't builds.

**Comment by GuillaumeGomez** [comment]
Hum, might be interesting to add it, indeed.

**Comment by GuillaumeGomez** [comment]
Updated.

File: release_process.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+ * Merge `master` branch into `crate` branch in every repository (starting with `sys` and ending with `gtk`). (/!\ `cairo-sys` is in `cairo` repository, not `sys` /!\)
+ * Update crate version in `Cargo.toml` and the number of its `gtk-rs` dependencies.
+ * Open pull requests to `crate` branches to corresponding repositories.
+ * Once pull requests have been merged, publish all crates to `crates.io` (using the `cargo publish` command).
+ * Generate new docs. To do so, go to `gtk` directory (in `crate` branch) and run `cargo doc --features=embed-lgpl-docs`. Then run `cp -r target/doc/* ../docs` (where `docs` is the corresponding directory for the `gtk-rs/docs` repository). Commit then push the changes to the `docs` (make a pull request and all the usual stuff...).
+ * Merge `pending` branch into `master` in the `gtk-rs/examples` repository (by opening a pull request of course).
+ * Update badges version number in the `_data/crates.json` in the `gtk-rs/gtk-rs.github.io` repositoryz.
+ * Write a blog post (add the file into `_posts` folder in `gtk-rs.github.io` repository) announcing the new release.
+
+NOTE: Pull requests on the `crate` branch aren't build.
```
Comment (EPashkin) [inline release_process.md]: Not good with english, but `builds` ?

**Comment by EPashkin** [comment]
Sorry, I done for today

**Comment by GuillaumeGomez** [comment]
No problem, thanks a lot for your help!

File: release_process.md (hunk 3 above)
Comment (GuillaumeGomez) [inline release_process.md]: No, in here, `build` is a verb.

**Pull request merged**